src/opm/parser/eclipse/EclipseState/Schedule/MSW/WellSegments.cpp

```diff
@@ -481,8 +459,7 @@ namespace Opm {
     bool WellSegments::updateWSEGVALV(const std::vector<std::pair<int, Valve> >& valve_pairs) {
 
         if (m_comp_pressure_drop == CompPressureDrop::H__) {
-            const std::string msg = "to use WSEGVALV segment for well " + m_well_name
-                                    + " , you have to activate the frictional pressure drop calculation";
+            const std::string msg = "to use WSEGVALV segment you have to activate the frictional pressure drop calculation";
```

Comment:
Here and above we lost some error information related to the well name.  No strong opinion, maybe it should be passed in though.

Reply:
Yes - I know; to simplify the construction of these objects from restart files I am *desperately* trying to reduce state. Probably not difficult to add well name back in at a later stage.